src/libsync/ownsql.cpp

```diff
@@ -303,6 +303,10 @@ void SqlQuery::bindValue(int pos, const QVariant& value)
                 res = sqlite3_bind_null(_stmt, pos);
             }
             break; }
+        case QVariant::ByteArray: {
+            auto ba = value.toByteArray();
+            res = sqlite3_bind_text(_stmt, pos, ba.constData(), ba.size(), SQLITE_TRANSIENT);
```

Comment:
I actually don't know what's the difference here.  I guess it's maybe related to \0 in the string?  Then maybe indeed blob could be better (more generic)  but currently, in owncloud, we don't have any field of type blob,  only text.